tests/pytorch/test_pytorch_autolog.py

```diff
@@ -70,7 +70,9 @@ def test_pytorch_autolog_ends_auto_created_run(pytorch_model):
 def pytorch_model_with_callback(patience):
     mlflow.pytorch.autolog()
     model = IrisClassification()
-    early_stopping = EarlyStopping(monitor="val_loss", mode="min", patience=patience, verbose=True)
+    early_stopping = EarlyStopping(
+        monitor="val_loss", mode="min", min_delta=99999999, patience=patience, verbose=True
+    )
```

Comment:
`min_delta=99999999` means `loss reduction less than 99999999 is not considered as an improvement` .

Reply:
The keras and fastai autologgng tests also use this approach.